src/constants/map.js

```diff
@@ -67,7 +67,7 @@ const map = {
       annotation.Measure,
   },
   rectangularAreaMeasurement: {
-    icon: 'ic_annotation_rectangular_area_black_24px',
+    icon: 'ic_annotation_ellipse_area_black',
```

Comment:
Hmm should we keep the original value? If we change this to `ellipse` then the icon in the notes panel of a rectangular polygon annotation will be changed to `ellipse`